database/dbtx.go

```diff
@@ -0,0 +1,86 @@
+package database
```

Comment:
Please also rename the file dbtx.go to something else.
I think the most fitting would be context.go, as it 99% deals with contexts.